clang/test/Driver/system_debug.c

```diff
@@ -0,0 +1,12 @@
+// Ensure -no-system-debug is passed to cc1 appropriately
+
+// RUN: %clang -fsystem-debug    -### -c %s 2>&1 | FileCheck %s --check-prefix=CHECK-POS
```

Comment:
nit: extra spaces between `-fsystem-debug` and `-###`

Reply:
Extra spaces removed.